.github/workflows/release.yml

```diff
@@ -0,0 +1,73 @@
+name: release
+
+on:
+  push:
+    tags:
+      - v*
```

Comment:
The job will run when any tag starting with `v` is pushed (including `v1.4.1-beta2`) to any branch, which is not what we want

Reply:
We can either define a pattern for tags of "solid" releases, that should be uploaded to PyPi, or additionally check if the tagged commit is in release branch (e.g. `master`). I suggest the first option, since Github Actions doesn't currently allow combining tag and branch checks in a convenient way. How about `v[0-9]\.[0-9]+\.[0-9]+`?

Reply:
Use `release` [event](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#release)